src/vconsole/vconsole-setup.c

```diff
@@ -55,7 +55,7 @@ static bool is_vconsole(int fd) {
 }
 
 static bool is_allocated(unsigned int idx) {
-        char vcname[strlen("/dev/vcs") + DECIMAL_STR_MAX(int)];
+        char vcname[sizeof("/dev/vcs") + DECIMAL_STR_MAX(int) - 2];
```

Comment:
we don#t actually mind if you call strlen() like this here. In theory it's a function call, and hence this would become a VLA. However, on gcc and llvm (and probably all smart compilers) strlen() on a literal string is optimized away, and constant. They grok that strlen() is what it is...

But using sizeof() is fine too